app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,13 @@
+  #comments
+    h2 Comments
+
+    - commentable.comments.each do |comment|
+      hr(id="#{dom_id comment}")
+      h5 = "#{ user_for_comment(comment) }, #{ l(comment.created_at, format: :short) }".html_safe
```

Comment:
Hm, maybe out of the scope of this PR, but I kinda don't like that we here first query for the comments and then **again** for the comment's user *(even though we in the end only want his or her name...)*. Do you think it would make sense to add something like a `comments_with_author` method to the concern used to make model commentable that would do a `includes(:user)`? One could then eventually also simplify the helper method used here a bit...